packages/web-react/.eslintrc.js

```diff
@@ -43,6 +43,23 @@ module.exports = {
     ],
     // problem of export default {} in stories
     'react/display-name': 'off',
+    // ignore UNSTABLE_ prefix in component names
+    'react/jsx-pascal-case': [
+      'error',
+      {
+        allowAllCaps: true,
```

Comment:
I am thinking about this. `allowAllCaps` means that `TEST_COMPONENT` is allowed. And `ignore` should say what is ignored by this. So shouldn't this be set to false? Like only `UNSTABLE_*` should be allowed...

Reply:
This is copypasted from the default config. I just needed to extend it to add the UNSTABLE part. 

AFAIK we use this https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/react.js#L132

But if you still think it should be false, I can change it :)

Reply:
Ok, let it be.